pdfminer/pdffont.py

```diff
@@ -180,92 +180,92 @@ def getdict(data):
                     value = b1 << 8 | b2
                 else:
                     value = b1 << 24 | b2 << 16 | \
-                            struct.unpack('>H', fp.read(2))[0]
+                        struct.unpack('>H', fp.read(2))[0]
```

Comment:
Only change indents on lines you actually change to keep git history meaningfull